package.json

```diff
@@ -134,6 +134,7 @@
     "stylelint-config-standard-scss": "^13.0.0",
     "tailwindcss": "^3.4.1",
     "ts-jest": "^29.1.2",
+    "ts-node": "^10.9.2",
```

Comment:
Given the script lives in `package/components`, this dev dep should actually belong there.

Reply:
Fair point 👍